langserver/workspace_refs.go

```diff
@@ -130,6 +130,11 @@ func (h *LangHandler) workspaceRefsTypecheck(ctx context.Context, bctx *build.Co
 		span.Finish()
 	}()
 
+	findPackage := h.FindPackage
+	if findPackage == nil {
+		findPackage = defaultFindPackageFunc
+	}
```

Comment:
@keegancsmith sorry I should've been more clear on Slack, I was preparing a PR that does exactly this among other things :)